doc/guix.md

```diff
@@ -0,0 +1 @@
+../contrib/guix/README.md
```

Comment:
29359b904f40805842447d4efa680bcc45bfdb57
Not sure if symlinking works in the GH GUI.

Reply:
Right, sorry, our tooling cannot handle this at the moment.
```
$ ../devtools/github-merge.py 20619
ERROR: File doc/guix.md was a symlink
```